Networking/NetworkingTests/Remote/ShippingLabelRemoteTests.swift

```diff
@@ -0,0 +1,40 @@
+import XCTest
+import TestKit
+
```

Comment:
nit
```suggestion
```

Reply:
removed in: https://github.com/woocommerce/woocommerce-ios/pull/3155/commits/562395447db31ea72d79aa601f15a7570421eec1